lib/content_item_publisher/finder_presenter.rb

```diff
@@ -0,0 +1,21 @@
+module ContentItemPublisher
+  class FinderPresenter < ContentItemPresenter
+    def present_links
+      links = {}
+      links["email_alert_signup"] = email_alert_signup if email_alert_signup?
```

Comment:
could make line 5 and 6 more consistent. - they do a similar thing but are implemented differently